### PR TITLE
Fix arguments to `preprocessCss` (to match classic build)

### PR DESCRIPTION
**Pull request by thoov**
In classic builds the second argument is '/app/styles' but we were passing 'app/styles'. While this difference might be small, in
ember-cli-eyeglass [it removes the leading slash](https://github.com/linkedin/eyeglass/blob/master/packages/ember-cli-eyeglass/src/index.ts#L210-L212) and this causes issues when the slash is missing. This PR aligns with what classic was doing along with adding a very simple test case.

Fixes: #908

### CODE REVIEW

**Review by rwjblue — APPROVED** [review]

**Comment by ef4** [comment]
CI is now back to normal, the remaining failures here are real failures, I think.

File: tests/fixtures/preprocess-addon/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+module.exports = {
+  name: require('./package').name,
+
+  setupPreprocessorRegistry(type, registry) {
+    registry.add('css', {
+      name: this.name,
+      ext: 'css',
+      toTree: (tree, inputPath, outputPath) => {
+        console.log('inputPath: ' + inputPath);
```
Comment (asakusuma) [inline tests/fixtures/preprocess-addon/index.js:32]: I assume you don't actually want to commit these console.logs in this file?

Comment (thoov) [reply tests/fixtures/preprocess-addon/index.js:32]: No I am trying to debug the windows path issue

**Comment by thoov** [comment]
@rwjblue This should be good to go

**Pull request merged**